pages/js/jquery.magnific-popup.js

```diff
@@ -1,3 +1,1175 @@
\ No newline at end of file
+(function (a) {
+  'use strict'
+  typeof define === 'function' && define.amd
+    ? define(['jquery'], a)
+    : typeof exports === 'object'
+      ? a(require('jquery'))
+      : a(window.jQuery || window.Zepto)
+})(function (a) {
+  'use strict'
+  const b = 'Close'
+  const c = 'BeforeClose'
+  const d = 'AfterClose'
+  const e = 'BeforeAppend'
+  const f = 'MarkupParse'
+  const g = 'Open'
+  const h = 'Change'
+  const i = 'mfp'
+  const j = '.' + i
+  const k = 'mfp-ready'
+  const l = 'mfp-removing'
+  const m = 'mfp-prevent-close'
+  let n
+  const o = function () {}
+  const p = !!window.jQuery
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/ruizivo/ahgora-login/pullRequest?prid=9873926)
```suggestion
  const p = !!window.jQuery;
```